src/Cake.Core/Cake.Core.csproj

```diff
@@ -131,11 +131,21 @@
     <Compile Include="IO\IRegistry.cs" />
     <Compile Include="IO\IRegistryKey.cs" />
     <Compile Include="IO\Machine.cs" />
+    <Compile Include="IO\NuGet\AssemblyFilePathFrameworkNameParser.cs" />
```

Comment:
Maybe I should do the same for FrameworkCompatibility too?
That would probably even include the `PackageReferenceSet`, `PackageReferenceBundler`, etc.